python/ray/rllib/agents/a3c/a3c.py

```diff
@@ -10,6 +10,7 @@
 from ray.rllib.utils import merge_dicts
 from ray.tune.trial import Resources
 
+# yapf: disable
```

Comment:
Why this change? It was passing `yapf` before, right?

Reply:
Yapf was adding newlines, messing up the doc includes.